src/RequestsLibrary/SessionKeywords.py

```diff
@@ -573,16 +576,14 @@ def _common_request(
             cookies=self.cookies,
             verify=self.verify,
             **kwargs)
-
         log.log_request(resp)
         self._print_debug()
-        session.last_resp = resp
```

Comment:
Just noticed that there was already a session variable where the last response was stored... not sure whether anyone knows or used it.

Reply:
it's ok, we can use that one :D